From 84b526fe22c25e6f95e0cf5fb683741c65b81a36 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Mon, 17 Aug 2015 15:19:48 +0800
Subject: [PATCH 1/3] powerpc/85xx: l2sram: include linux/of_address.h

Fix the following build error:
  arch/powerpc/sysdev/fsl_85xx_l2ctlr.c: In function 'mpc85xx_l2ctlr_of_probe':
  arch/powerpc/sysdev/fsl_85xx_l2ctlr.c:105:2: error: implicit declaration of function 'of_iomap' [-Werror=implicit-function-declaration]
    l2ctlr = of_iomap(dev->dev.of_node, 0);
    ^
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/sysdev/fsl_85xx_l2ctlr.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c b/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
index 5d7384c..3b9d33e 100644
--- a/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
+++ b/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
@@ -23,6 +23,7 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/of_platform.h>
+#include <linux/of_address.h>
 #include <asm/io.h>
 
 #include "fsl_85xx_cache_ctlr.h"
-- 
1.7.5.4

